lib/postgresql/native_postgres.nit

```diff
@@ -83,27 +85,27 @@ end
 extern class NativePostgres `{PGconn *`}
 
   # Connect to a new database using the conninfo string as a parameter
-  new connectdb(conninfo: String) import String.to_cstring `{
+  new connectdb(conninfo: Text) import String.to_cstring `{
     PGconn * self = NULL;
     self = PQconnectdb(String_to_cstring(conninfo));
```

Comment:
In pure theory this should have be detected by the C compiler. But I haven't correctly implemented it in our "new" compiler, I'll have to update it.